man/zypper-docker-up.1.md

```diff
@@ -0,0 +1,36 @@
+% ZYPPER-DOCKER(1) zypper-docker User manuals
+% SUSE LLC.
+% SEPTEMBER 2015
+# NAME
+zypper\-docker update \- Install the available updates for the given image.
+
+# SYNOPSIS
+**zypper-docker update** [command options] IMAGE NEW-IMAGE
+
+# DESCRIPTION
+The **update** command updates the given openSUSE/SLES image with all the
+available updates. The updated image will have a new name, as provided by the
+NEW-IMAGE argument.
+
```

Comment:
I think we should write down IMAGE and NEW-IMAGE are using docker's format. That was specified elsewhere.